app/src/gecko/java/org/mozilla/focus/web/GeckoViewPrompt.java

```diff
@@ -688,8 +710,9 @@ public void onClick(final DialogInterface dialog, final int which) {
         createStandardDialog(builder, callback).show();
     }
 
-    public void promptForFile(GeckoSession session, String title, int type,
-                              String[] mimeTypes, FileCallback callback) {
+
+    @Override
+    public void onFilePrompt(GeckoSession session, String title, int type, String[] mimeTypes, FileCallback callback) {
```

Comment:
\[PMD\] INFO: The method 'onFilePrompt' has a Standard Cyclomatic Complexity of 11\.